src/WorkerHandler.js

```diff
@@ -220,13 +220,15 @@ function objectToError (obj) {
 
 function handleEmittedStdPayload(handler, payload) {
   // TODO: refactor if parallel task execution gets added
-  if (Object.keys(handler.processing).length !== 1) {
```

Comment:
I can add back the length check, but it was removed because Object.values returns an empty array if the object is empty. When accessing elements in an array, if the index is not defined accessing will return undefined. So my addition of a check on the task being defined should suffice for both cases.

Reply:
I can do this refactor, the current code will do less iteration but outside of that i dont see a reason to not use the pattern you're recommending.

This will rely on the current architecture model to assure there is only ever a single task in `processing` or `tracking` but I think that is fine as we know each worker only is assigned a single task at a time.